### DISCUSSION TITLE
Sonoff T0 Ver 2 Flashing

### DISCUSSION THREAD

**Discussion by rg2k** [arendst/Tasmota#15234] [Q&A] [answered]
Hi All,

The Sonoff T0 PCB has changed significantly between Ver 1 and 2, where would I find GPIO0 now?

Thanks
![IMG_1183](https://user-images.githubusercontent.com/6370611/160271563-5b83bcae-2c31-457b-90e6-b9ac94ff8446.jpeg)

**Comment by sfromis** [comment, marked as answer]
Instead of looking for a place where they've "hidden" it on the board, simplest is to ground it directly on the ESP8285 chip, [pin](https://tasmota.github.io/docs/Pinouts/) 15 along with the neighbor pin 16. When orienting the chip (and whole board) for reading the labels (90 degrees clockwise from your image), it is the two right-hand pins on the bottom row of tiny pins on the chip.

The only reason for me suggesting pin 16 also is that it makes it much easier when you can rest a pin between them, instead of being very careful to touch only one, and it does not do any harm to also ground gpio4 at the same time as gpio0.

**Reply by rg2k** [reply]
@sfromis, you are a genius. That worked, thanks so much!
